### PR TITLE
fix type and default for retention_local_trim_interval

### PR DESCRIPTION
**Pull request by Deflaimun**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2792
Review deadline:

## Page previews

https://deploy-preview-880--redpanda-docs-preview.netlify.app/current/reference/properties/cluster-properties/

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [x] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/reference/pages/properties/cluster-properties.adoc [hunk 1]
```diff
@@ -4451,7 +4451,7 @@ The period during which disk usage is checked for disk pressure, and data is opt
 
 *Accepted values:* [`-17592186044416`, `17592186044415`]
```
Comment (Deflaimun) [inline modules/reference/pages/properties/cluster-properties.adoc]: Should we make the accepted values start from 1? 
(for some duration properties, negative duration is accepted)

Comment (Deflaimun) [inline modules/reference/pages/properties/cluster-properties.adoc]:
```suggestion
*Accepted values:* [`1`, `17592186044415`]
```

Comment (Deflaimun) [reply modules/reference/pages/properties/cluster-properties.adoc]: Changed to 1

File: modules/reference/pages/properties/cluster-properties.adoc [hunk 2]
```diff
@@ -4449,9 +4449,9 @@ The period during which disk usage is checked for disk pressure, and data is opt
 
 *Type:* integer
 
-*Accepted values:* [`-17592186044416`, `17592186044415`]
+*Accepted values:* [`1`, `17592186044415`]
```
Comment (Deflaimun) [inline modules/reference/pages/properties/cluster-properties.adoc]:
```suggestion
*Accepted values:* [`0`, `17592186044415`]
```

**Review by nvartolomei — APPROVED** [review]

**Review by asimms41 — APPROVED** [review]
LGTM

**Pull request merged**